add-domain

```diff
@@ -522,3 +522,4 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
```

Comment:
Can only see a default Apache Centos page. Do you have any other information for confirmation?